manager/examples/activated_probes/ebpf/include/bpf_helpers.h

```diff
@@ -0,0 +1,151 @@
+#ifndef __BPF_HELPERS_H
```

Comment:
nit: It would be nice if we had only one copy of these common headers for all the examples. It increases the size of the repo.

Reply:
Agreed, I'll make a PR to change all the examples at the same time.